sample/flask-of-ziti/helloFlazk.py

```diff
@@ -19,20 +19,18 @@
 app = Flask(__name__)
 bind_opts = {}  # populated in main
 
-
-@openziti.zitify(bindings={
-    ':18080': bind_opts,
-})
+@openziti.zitify(bindings={':18080': bind_opts,})
 def runApp():
     from waitress import serve
+    print("starting server on OpenZiti overlay")
+    #the port is only used to integrate OpenZiti with frameworks that expect a "hostname:port" combo
     serve(app,port=18080)
 
-
 @app.route('/')
 def hello_world():  # put application's code here
+    print("received a request to /")
```

Comment:
maybe better to enable Flask access log

Reply:
Here's how I adapted this sample with logging for the text classifier API: https://github.com/openziti-test-kitchen/ziti-flask-api-text-classifier/blob/main/ziti-classifier-api.py#L49